.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,14 @@
+### Description of changes:
+
+### CTSM Issues Fixed (include github issue #):
```

Comment:
On other projects, we've used something like the following template: 

```
 - [ ] Closes #xxxx (remove if there is no corresponding issue, which should only be the case for minor changes)
 - [ ] Tests added (for all bug fixes or enhancements)
 - [ ] Tests passed (for all non-documentation changes)
 - [ ] Fully documented, ...)
```

This seems to help new/old developers actually provide this information.